src/components/ec-timer/ec-timer.vue

```diff
@@ -93,43 +96,44 @@ export default {
     offset() {
       return this.circumference + this.steps * this.secondsLeft;
     },
-    timeDifference() {
-      return this.currentTime - this.startTime;
-    },
   },
   watch: {
     isRunning: {
       immediate: true,
       handler(value) {
         if (value) {
-          this.countdown();
+          this.startCountdown();
         } else {
-          clearInterval(this.timerInterval);
+          if (this.countdown) {
+            this.countdown.stop();
```

Comment:
Including `this.countdown = null;` also here we avoid to invoke the `this.countdown.stop();` again the next time `isRunning` is set to `true`

Reply:
Done!